pkg/services/accesscontrol/filter.go

```diff
@@ -84,25 +85,44 @@ func Filter(user *models.SignedInUser, sqlID, prefix string, actions ...string)
 	return SQLFilter{query.String(), ids}, nil
 }
 
-func parseScopes(prefix string, scopes []string) (ids map[int64]struct{}, hasWildcard bool) {
-	ids = make(map[int64]struct{})
+func parseScopes(prefix, attribute string, scopes []string) (ids map[interface{}]struct{}, hasWildcard bool) {
+	ids = make(map[interface{}]struct{})
+	scopePrefix := Scope(prefix, attribute)
+	parser, ok := scopeParsers[attribute]
```

Comment:
question: would it make sense to return `nil, false` if there is no scope parser defined for the attribute?